src/main/cpp/t04_count.cpp

```diff
@@ -28,5 +28,15 @@
 using namespace std;
 
 int t04_count() {
+int q = 2;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'q' can be reduced.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4494122)